docs/available-commands.md

```diff
@@ -212,6 +216,18 @@ Command to copy text directly from the URL to the clipboard. Handy for short str
 * Input: URL encoded text to copy onto the clipboard
 * Example: `didact://?commandId=vscode.didact.copyToClipboardCommand&text=The%20fox%20jumped%20over%20the%20lazy%20dog.`
 
+## vscode.didact.copyTextToCLI
+
+Command to take the currently selected text in the open editor (of a Didact Markdown or AsciiDoc file) and add a new Didact `sendNamedTerminalAString` or `sendNamedTerminalAStringNoLF` link with the selected text automatically URLencoded.
+
+This is triggered by default using `Ctrl+Alt+F` or `Cmd+Alt+F`.
+
+A couple of things to note:
+
+* Any link inserted using this method will be surrounded by parentheses. For example, `^ execute` will appear in the Didact link as `(^ execute)`. 
```

Comment:
What about putting the parenthesis in the default value instead of inserting them always?
it will avoid to force users to use parenthesis for their link. For intance to use { or [, or even no specific markers and just an emoji.

Reply:
The parens are not included in the default value to avoid them being part of the link itself. It is easier to see linked text when the parens are NOT in the link itself.

Reply:
ok, I see the use case.
So my thinking would be more to have the option to customize these parens. Which is completely overkill in a this iteration. And maybe even in another one.